docs/vcpkg-schema-definitions.schema.json

```diff
@@ -915,7 +929,7 @@
         {
           "not": {
             "description": "Identifiers and parts delimited by slashes must not be a Windows filesystem or vcpkg reserved name.",
-            "pattern": "(^|/)(prn|aux|nul|con|lpt[1-9]|com[1-9]|core|default)(/|$)"
+            "pattern": "(^|/)(prn|aux|nul|con|lpt[0-9]|com[0-9]|core|default)(/|$)"
```

Comment:
This change needs corresponding product code changes, and a test:

https://github.com/microsoft/vcpkg-tool/blob/c5e532a1a33943f122fcd18ed0e1df5a4ec68ed4/src/vcpkg/base/json.cpp#L1081-L1089

Reply:
[src/vcpkg/base/json.cpp#L1056](https://github.com/microsoft/vcpkg-tool/blob/2024-06-10/src/vcpkg/base/json.cpp#L1056) `is_ident`
[src/vcpkg-test/registries.cpp#L114](https://github.com/microsoft/vcpkg-tool/blob/2024-06-10/src/vcpkg-test/registries.cpp#L114) `check valid package patterns`
~WIP~